packages/components/src/components/primitives/Button/Button.stories.tsx

```diff
@@ -40,18 +41,39 @@ const AllTemplate = () => (
             </Button>
           ))}
         </HStack>
+        <HStack key={variant}>
+          {ButtonSizes.map((size) => (
+            <Button
+              variant={variant}
+              size={size}
+              key={size}
+              trailingIcon={Icons.IconExternalLink}
+              leadingIcon={Icons.IconExternalLink}
+            >
+              {variant}
+            </Button>
+          ))}
+        </HStack>
       </VStack>
     ))}
   </VStack>
 );
 
 export const All = AllTemplate.bind({});
 
-const Template = (args) => <Button {...args}></Button>;
+const Template = (args) => (
+  <Button
+    {...args}
```

Comment:
`showLeadingIcon` and `showTrailingIcon` shouldn't be passed to the component

Reply:
fixed